contributor_path.rst

```diff
@@ -0,0 +1,212 @@
+****************
+Contributor path
+****************
+
+..
+
+  "There is no favorable wind for the sailor who doesn’t know where to go."
+
+  -- Seneca
+
+This guide describes a contributor path in scope of the Ansible project from the early beginning, through all the stages, to becoming a leader shaping the project's future and the future of automation in the IT world in general.
+
+This path is not a policy but rather a suite of recommendations based on experience of successful contributors.
+
+The goal of the path, first of all, is to make your activity more determined and efficient through understanding of what you can achieve and how. You can use this path as a roadmap for your long-term participation.
+
+Any contribution to the project, even a small one, is very welcome and valuable. When you contribute regularly, your proficiency and judgement in the related area are increasing fast and, along with this, the importance of your presence in the project.
+
+Regular contribution makes you a member of the community and leads to respect of other community members.
+
+Be persistent and act gradually. Great result is just a product of time and effort. Here's how to get started.
+
+.. note::
+
+   If you have any questions, feel free to reach out to the community in the ``#ansible-community`` `Libera.Chat <https://libera.chat/>`_ IRC channel. If you have ideas on how this document can be improved, please, share them through creating an issue or a pull request.
+
+.. contents::
+
+What is a contribution to an Open Source project
+================================================
+
+First, we should understand what a contribution to an Open Source project is.
+
+The contribution to an Open Source project is any positive addition or change given by an individual or a company to the project including increasing its functionality, improvements to the codebase, code developer experience, user experience, increasing the project's popularity and community, providing infrastructure, and any other kinds of active social or material support.
```

Comment:
```suggestion
Contributing to an Open Source project can be any positive addition, change or feedback given by an individual (or company) to the project. It includes increasing its functionality, improving the codebase as well as developer experience and user experience, promoting the project and participating in the community, providing infrastructure, and any kind of active social or material support.
```

Can't pretend to be a wordsmith. Does the above sound better ?

Reply:
I like it! Committed. I only left `The contribution to an Open Source project is any positive addition` (the part before the first comma) as it was because (IMO) when we define a notion, we should use the pattern `noun + is`, though I'm not an expert.
Thanks for the rephrasing!